### DISCUSSION TITLE
my students educational copilot，why?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#120534] [GitHub Education]
### Hi everyone,

![2 BB~C$Q9`13~O$54W`YQP5](https://github.com/community/community/assets/106868238/ae9ddb52-7dfc-427d-8b9b-34448332b824)

**Comment by ettaboyle** [comment]
Hi @RobbenZhu, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
